src/main/java/uk/gov/companieshouse/api/strikeoffobjections/file/FileTransferApiClient.java

```diff
@@ -0,0 +1,133 @@
+package uk.gov.companieshouse.api.strikeoffobjections.file;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Component;
+import org.springframework.util.LinkedMultiValueMap;
+import org.springframework.web.client.RestTemplate;
+import org.springframework.web.multipart.MultipartFile;
+import uk.gov.companieshouse.api.strikeoffobjections.common.ApiLogger;
+
+import java.io.IOException;
+
+@Component
+public class FileTransferApiClient {
+
+    private static final String HEADER_API_KEY = "x-api-key";
+    private static final String UPLOAD = "upload";
```

Comment:
It was probably already like this in extensions, but I think UPLOAD_FILE_FIELD_NAME = "upload" would be more meaningful

Reply:
Changed name.